src/scripts.js

```diff
@@ -11,22 +12,65 @@ let userDailyStepGoal = document.querySelector('.user-daily-step-goal-js');
 let userFriends = document.querySelector('.user-friends-js');
 let stepGoalComparison = document.querySelector('.step-goal-comparison-js');
 let waterConsumedByDay = document.querySelector('.water-consumed-by-day-js');
-let waterConsumedByWeek = document.querySelector('.water-consumed-by-week-js')
+let waterConsumedByWeek = document.querySelector('.water-consumed-by-week-js');
```

Comment:
adding querySelectors to populate the different tiles on the Dashboard based on where we want the data to appear.